### DISCUSSION TITLE
ABSOLUTE结果解读相关

### DISCUSSION THREAD

**Discussion by Liyang-Zhang** [ShixiangWang/self-study#109] [Q&A] [answered]
翔哥好！

我目前在进行WES的SNVs结果克隆性位点和亚克隆位点分类的研究，在我读到ABSOLUTE的文章后发现应该就是使用这个软件进行分类。

我现在还有一些疑惑：

1. 我使用官方的测试数据得到了ABS_MAF文件，这里面的列很多，部分列的信息我不太清楚代表什么，我也没有google到相关的manual。我想请问下翔哥你那有解释那些列的网站或者文档不
2. 对于得到的CCF的可能性的计算过程翔哥你有弄清楚其中的计算过程吗。我有读到一些文章都是使用了类似的方法（包括ABSOLUTE的结果文件），但我看不明白，有点懵。
<img width="558" alt="300e6e78c2a630407a6c632ea3a40ee" src="https://user-images.githubusercontent.com/72248852/199414528-3eac93ee-b9df-4749-ba09-2601e8777399.png">

Bw,
liyang

**Comment by ShixiangWang** [comment, marked as answer]
我没有研究过ABSOLUTE结果MAF，不知道它跟普通的有什么区别，关于 MAF 你可以看看 https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/

ABSOLUTE 结果解读很久之前做过笔记，看下 https://www.jianshu.com/p/388fb14989df

**Reply by ShixiangWang** [reply]
计算公式参考 https://shixiangwang.github.io/home/cn/post/2018-07-24-absolute-inference/ 这个理解

**Reply by Liyang-Zhang** [reply]
好的，感谢

**Reply by Liyang-Zhang** [reply]
翔哥，我再想请教下关于ABSOLUTE文档的问题。ABSOLUTE documentation的链接网址已经失效了http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/ABSOLUTE

然后我也没有检索到其他官方的软件说明文档。请问你之前有保存过ABSOLUTE的manual吗，或者有他的说明文档的其他网址不

**Reply by ShixiangWang** [reply]
> 翔哥，我再想请教下关于ABSOLUTE文档的问题。ABSOLUTE documentation的链接网址已经失效了http://software.broadinstitute.org/cancer/software/genepattern/modules/docs/ABSOLUTE
> 
> 然后我也没有检索到其他官方的软件说明文档。请问你之前有保存过ABSOLUTE的manual吗，或者有他的说明文档的其他网址不

https://software.broadinstitute.org/cancer/cga/absolute_run

**Comment by yzhwzh** [comment]
翔哥好，请问下，你那里还有ABSOLUTE的原代码安装包吗，现在网上搜不到了

**Reply by ShixiangWang** [reply]
https://github.com/ShixiangWang/DoAbsolute/tree/master/inst/extdata 这里有存档，希望对你有帮助
